diffeqpy/packaging.jl

```diff
@@ -0,0 +1,4 @@
+function install()
```

Comment:
I put this in a function so that it's easier to do Julia specific thing. For example, in Julia 1.0 you'd want `using Pkg` (or is it just warning?)

Reply:
I think `Pkg` operations will still work, so this is good.